storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3656,8 +3668,17 @@ static int rocksdb_init_func(void *const p) {
       (rocksdb::BlockBasedTableOptions::IndexType)rocksdb_index_type;
 
   if (!rocksdb_tbl_options->no_block_cache) {
-    rocksdb_tbl_options->block_cache =
+    std::shared_ptr<rocksdb::Cache> block_cache =
         rocksdb::NewLRUCache(rocksdb_block_cache_size);
+    if (rocksdb_sim_cache_size > 0) {
```

Comment:
Ah, nevermind. I missed the fact the NewSimCache is constructed from the newly allocated LRU block_cache. I thought it was a standalone cache type, hence, the thought on expanding this to support different types of caches: LRU, FIFO, etc.